source/sync/data-model/update-data-model/update-schema-production-apps.txt

```diff
@@ -1,20 +1,430 @@
+
+   .. step:: Confirm and Deploy Changes
+
+      From the confirmation window, review the :guilabel:`Schema Version` and
+      :guilabel:`Change Summary` information:
+
+      - The current schema version, and the incremented number of the
+        new schema created upon deployment (if applicable).
+      - The type of change, labeled as ADDITIVE, SUPPORTED, or UNSUPPORTED.
+      - A description of the changes to the noted object type or field.
+      - Any notes or suggested actions to take for a given change. For example,
+        if App Services is expecting a default value based on the change you
+        made, the note suggests that you define a default value.
+
+      How you deploy your changes depends on the changes in your changeset:
+
+      .. tabs::
+
+         .. tab:: Schema Versioning
+            :tabid: supported
+
+            If your changeset includes at least one schema versioning change,
+            then all changes (including any non-breaking changes) are applied
+            to a new compatible schema version.
+
+            1. Review your changes, which may include non-breaking changes. We
+               recommend that you resolve any suggested actions noted before you
+               deploy your changes.
+
+            2. Click :guilabel:`Deploy`.
+
+            3. Your changes are applied to the new schema version. The new
+               version is compatible with previous versions.
+
+         .. tab:: Breaking
+            :tabid: unsupported
+
+            If your changeset includes at least one breaking change, then all
+            changes (including any non-breaking and schema versioning changes)
+            are applied in a single changeset. Upon deployment of breaking
+            changes, App Services terminates and re-initializes Device Sync.
+
+            1. Review your changes, which may include non-breaking and schema
+               versioning changes. We recommend that you resolve any suggested
+               actions noted before you deploy your changes.
+
+               Breaking changes may or may not increment the schema version:
+
+               - If the current schema version is greater than 0, App Services
+                 indicates that the schema will increment upon deployment.
+               - If the current schema version is 0, the window only displays
+                 the current schema version number.
+
+            2. Click :guilabel:`Deploy and re-initialize`.
+
+            3. After Device Sync terminates and re-initializes, your changes are
+               applied in one of two ways:
+
+               - If App Services created a new schema, it becomes the new
+                 minimum version and is incompatible with any previous schemas.
+               - If App Services did not create a new schema, the changes are
+                 applied to the current schema version 0.
+
+   .. step:: Handle Client Resets
```

Comment:
I'm wondering if we should make this step title more general, because don't we _have_ to update the version number in the client app with the new schema version in either case? I don't see that covered here. 

```suggestion
   .. step:: Update the Client
```